pygmt/src/meca.py

```diff
@@ -211,6 +215,20 @@ def meca(
         to the circle. Use **+s**\ *size* to set the diameter of the circle
         [Default is no circle]. Use **+p**\ *pen* to set the line pen
         attributes [Default is 0.25p].
+    compressionfill : str
+        Set color or pattern for filling compressive quadrants
+        [Default is black].
+    extensionfill : str
+        Set color or pattern for filling extensive quadrants
+        [Default is white].
+    pen : str
+        Set pen attributes for outline of beachball
+        [Default is ``"0.25p,black,solid"``].
+    cmap : str
+        File name of a CPT file or a series of comma-separated colors (e.g.,
+        *color1,color2,color3*) to build a linear continuous CPT from those
+        colors automatically. The color of the compressive quadrants is
+        determined by the z-value (e.g., event depth) in the third column.
```

Comment:
I wonder if the following changes are better.
```suggestion
        determined by the z-value (i.e., event depth or the third column for 
        an input file).
```

Reply:
For an input file or an array, z-value is the third columns.
For dict and pd.DataFrame, z-value is the event depth.